src/main/java/aquality/appium/mobile/elements/actions/ElementTouchActions.java

```diff
@@ -0,0 +1,122 @@
+package aquality.appium.mobile.elements.actions;
+
+import aquality.appium.mobile.actions.SwipeDirection;
+import aquality.appium.mobile.actions.TouchActions;
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.configuration.ISwipeConfiguration;
+import aquality.appium.mobile.elements.interfaces.IElement;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+
+public class ElementTouchActions implements IElementTouchActions {
+
+    private final Point scrollDownStartPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeBottomPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeBottomPointYCoefficient());
+
+    private final Point scrollDownEndPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeTopPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeTopPointYCoefficient());
+
+    private final Point swipeLeftStartPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeRightPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeRightPointYCoefficient());
+
+    private final Point swipeLeftEndPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeLeftPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeLeftPointYCoefficient());
+
+    private final Point scrollUpStartPoint = scrollDownEndPoint;
+
+    private final Point scrollUpEndPoint = scrollDownStartPoint;
+
+    private final Point swipeRightStartPoint = swipeLeftEndPoint;
+
+    private final Point swipeRightEndPoint = swipeLeftStartPoint;
+
+    @Override
+    public void swipe(IElement element, Point endPoint) {
```

Comment:
since the element persist in every method of this class, it should be moved to the constructor of this class.
We should also be able to getTouchActions() from IElement

Reply:
Apparently, there is no need in `getTouchActions()` in `IElement`, as it will be accessible from `AqualityServices`. What actually means that we will have 
```
default ITouchActions getTouchActions() {
        return AqualityServices.getTouchActions();
}
``` in the `IElement` interface. What do you think?

Reply:
Is the same true for the `TouchActions` class?

Reply:
oh no, there are two different touch actions. One would be accessible either from `AqualityServices.getTouchActions();` or from `AqualityServices.Application.getTouchActions();`.
Another one, ElementTouchActions, to be returned by Element's method:
```
public interface IElement extends aquality.selenium.core.elements.interfaces.IElement {
...
IElementTouchActions getTouchActions();
```    
```
public abstract class Element extends aquality.selenium.core.elements.Element implements IElement {
...
@Override
    public IElementTouchActions getTouchActions() {
             return new ElementTouchActions(this);
    }
```